src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArr = sourceString.split(';');
+  const EXPECTED_OBJECT = {};
+
+  sourceArr.forEach(e => {
+    const index = e.indexOf(':');
+    const key = e.slice(0, index).trim();
+    const value = e.slice(index + 1).trim();
+
+    if (key.length > 0) {
```

Comment:
0 converts to false and better to check value too

```suggestion
    if (key.length) {
```